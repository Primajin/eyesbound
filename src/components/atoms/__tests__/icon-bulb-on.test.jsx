import React from 'react';
import {create} from 'react-test-renderer';

import BulbOn from '../icons/bulb-on.jsx';

describe('BulbOn', () => {
	it('renders correctly', () => {
		const component = create(<BulbOn/>);
		expect(component).toMatchSnapshot();
	});
});
