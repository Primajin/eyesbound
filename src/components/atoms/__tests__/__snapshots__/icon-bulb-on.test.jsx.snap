// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`BulbOn renders correctly 1`] = `
<svg
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M0 0h24v24H0z"
    fill="none"
  />
  <path
    d="M9 21c0 .5.4 1 1 1h4c.6 0 1-.5 1-1v-1H9v1zm3-19a7 7 0 00-4 12.7V17c0 .5.4 1 1 1h6c.6 0 1-.5 1-1v-2.3A7 7 0 0012 2z"
  />
</svg>
`;
